app/models/user.js

```diff
@@ -48,6 +48,43 @@ const User = new mongoose.Schema({
     lowercase: true,
     unique: true,
     validate: val => validator.isEmail(val)
+  },
+  full_email: {
+    type: String,
+    required: true,
+    trim: true
+  },
+
+  // api token for basic auth
+  api_token: {
+    type: String,
+    required: true,
+    lowercase: true,
+    trim: true,
+    unique: true,
+    index: true
+  },
+
+  // password reset
+  reset_token_expires_at: Date,
+  reset_token: String,
+  has_set_password: {
+    type: Boolean,
+    default: false
+  },
+  
+  two_factor_enabled: {
```

Comment:
`two_factor_enabled` and `two_factor_token` should be put into the config (e.g. look at how `full_email` and `api_token` are defined in this user model based off config property name stored)